quiz.txt

```diff
@@ -0,0 +1,22 @@
+#4. Create salary.csv file which contain sid,sname,salary.
+  #Display the employee record whose name begins from "S? also show no. of employee with first letter "S? out of total record.
+
+import csv
+a=open('c:\\sqlite3\\salary.csv','w',newline='')
+b=csv.writer(a)
+c=['sid','sname','salary']
+b.writerow(c)
+d=[[1,'om',10000],[2,'sai',50000],[3,'ram',20000],[4,'umashiv',30000],[5,'radekrishna',20000]]
+b.writerows(d)
+e=open('salary.csv','r')
+f=csv.reader(e)
+for i in f:
+    print(i)
+
+output:
+['sid', 'sname', 'salary']
+['1', 'om', '10000']
+['2', 'sai', '50000']
+['3', 'ram', '20000']
+['4', 'umashiv', '30000']
+['5', 'radekrishna', '20000']
```

Comment:
where is a output of  the employee record whose name begins from "S" and also show no. of employee with first letter S

Reply:
sorry i can't understand the logic of this program so i put it only